openpower/package/openpower-pnor/Config.in

```diff
@@ -77,6 +77,18 @@ config BR2_WOFDATA_BINARY_FILENAME
         help
             String used to define name of wofdata binary ecc'd file
 
+config BR2_MEMDDATA_FILENAME
+        string "Name of memd original file"
+        default "memd_extra"
+        help
+            String used to define name of memd original file
+
+config BR2_MEMDDATA_BINARY_FILENAME
+        string "Name of memd binary file"
+        default "memd_extra_data.bin.ecc"
```

Comment:
why isn't it memd_extra.bin.ecc

or instead, above that, memd_extra_data.bin?

I have never really understood having to specify both config options since usually, there is an understanding that given an original *.bin filename, and if you add a header you get *.header.bin and if you add ECC you get *.header.bin.ecc

But, I guess this is a prevailing practice, so if no issue until now, no point in changing the approach.